cmd/kubeadm/app/images/images_test.go

```diff
@@ -215,23 +229,63 @@ func TestGetAllImages(t *testing.T) {
 					Local: &kubeadmapi.LocalEtcd{},
 				},
 			},
-			expect: constants.Etcd,
+			expectedImages: []string{
+				"/kube-apiserver:",
+				"/kube-controller-manager:",
+				"/kube-scheduler:",
+				"/pause:" + constants.PauseVersion,
+				"/etcd:" + constants.DefaultEtcdVersion,
+				"/coredns:" + constants.CoreDNSVersion,
+				"/kube-proxy:",
+			},
+		},
+		{
+			name: "CoreDNS and kube-proxy image is returned",
+			cfg:  &kubeadmapi.ClusterConfiguration{},
```

Comment:
add one testcase to have `Disabled` set to `false` for the two addon and get back them in the list of `expectedImages`.

Reply:
Done.